ruby/dalli/test/functional/posts_controller_test.rb

```diff
@@ -3,6 +3,16 @@
 class PostsControllerTest < ActionController::TestCase
   setup do
     @post = posts(:one)
+    set_http_auth
+  end
+
+  def set_http_auth
```

Comment:
These tests weren't working because they weren't setting authentication credentials. This fixes them to work with basic auth.